cmd/install/assets/hypershift_operator.go

```diff
@@ -439,12 +465,13 @@ func (o HyperShiftOperatorClusterRole) Build() *rbacv1.ClusterRole {
 					"namespaces",
 					"serviceaccounts",
 					"services",
+					"endpoints",
 				},
 				Verbs: []string{"*"},
 			},
 			{
-				APIGroups: []string{"apps"},
-				Resources: []string{"deployments"},
+				APIGroups: []string{"apps", "extensions"},
```

Comment:
Why do we need extensions?

Reply:
I don't think extenstions is required so I could also remove extensions from that list

Reply:
ah okay good, I was concerned that we maybe accidentally accessed stuff from the extensions api which would have meant we have duplicate caches

Reply:
I don't think we need `extensions` in that role. We should probably remove it in a followup.